src/components/comment.vue

```diff
@@ -23,11 +23,23 @@
         </ion-chip>
       </div>
     </ion-col>
-    <ion-col size="1">
+
+    <ion-col size="2" class="reply-chip" >
       <ion-chip @click="showInput = !showInput" outline color="light">
         <ion-icon :icon="replyIcon" size="small"/>
       </ion-chip>
     </ion-col>
+
+  </ion-row>
+
+
+<ion-row v-if="comment.replyTo">
+    <ion-col offset="2" size="10" class="comment-row">
+      {{text}} <avatar :profile="author" class="comment-avatar" />  <div class="authorName" v-if="comment.replyTo"> <div class="OnlyAutherName">{{authorName}}</div> &nbsp;<ion-note color="medium">{{since}}</ion-note></div>
```

Comment:
1. `v-if="comment.replyTo"` is already assured
2. `<avatar` has a name-feature, just add `with-name :name="authorName"` instead of adding custom divs. The only reason that was split before was because it was in different columns. I suspect after this change most css here is also unnecessary.